ui/src/Pages/AlbumPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router";
+import NavBar from "../components/NavBar";
+import { Tracks } from "../components/Tracks";
+import { Track, Album } from "../types";
+import PlayCircleRoundedIcon from "@mui/icons-material/PlayCircleRounded";
+import { green, red } from "@mui/material/colors";
+import FavoriteBorderOutlinedIcon from "@mui/icons-material/FavoriteBorderOutlined";
+import Auth from "../utils/Auth";
+
+export default function AlbumPage() {
+  const { id } = useParams();
+  const [album, setAlbum] = useState({
+    name: "",
+    image: "",
+    artist: "",
+    date: "",
+    total: "",
+  });
+  const [albumTracks, setAlbumTracks] = useState<Track[]>([]);
+  const accessToken = Auth();
+
+  const getAlbum = () => {
+    const searchParameters = {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${accessToken}`,
```

Comment:
Use Proper flow for using accessTokens from the parameters instead of the state directly especially when it is important.